pkg/lib/authn/user/store.go

```diff
@@ -95,6 +96,32 @@ func (s *Store) Get(userID string) (*User, error) {
 	return u, nil
 }
 
+func (s *Store) GetByIDs(userIDs []string) ([]*User, error) {
+	ids := make([]interface{}, len(userIDs))
+	for i, id := range userIDs {
+		ids[i] = id
+	}
+	builder := s.selectQuery().
+		Where("id IN ("+squirrel.Placeholders(len(ids))+")", ids...)
```

Comment:
I remember I have done similar thing before. I used postgresql array that time so there is only one placeholder in the query.

Reply:
I copied this from the only place using it (password history). Maybe it is lost with time already?

Let me double check if we can use pg array instead.

Reply:
It turns out the syntax you used is `id = ANY (?)` so I didn't found it...

I'll fix everywhere to use that instead.